Sources/MongoSwift/Operations/Operation.swift

```diff
@@ -94,13 +132,6 @@ internal class OperationExecutor {
     }
 }
 
-/// Given a client and optionally a session associated which are to be associated with an operation, returns a
-/// connection for the operation to use. After the connection is no longer in use, it should be returned by
-/// passing it to `returnConnection` along with the same client and session that were passed into this method.
-internal func resolveConnection(client: MongoClient, session: ClientSession?) throws -> Connection {
```

Comment:
we only called this method in one place, and also its docstring was out of date. it seemed better to just move this logic into the place we called this, where it was already documented